hack/hypernetes/ceph.sh

```diff
@@ -20,10 +20,24 @@ set -o pipefail
 
 function kube::util::setup_ceph() {
 	echo "Start $FUNCNAME"
+	rm -rf /etc/ceph
+	mkdir -p /etc/ceph
+	if ! kube::util::do_setup_ceph ; then
+		echo "Preinstall some package before exectue ceph-deploy..."
+		kube::util::ceph_preinstall
+		kube::util::do_setup_ceph
```

Comment:
I think just calling one `kube::util::do_setup_ceph` is enough here. If the install is failed, the script could be run again.